codility_training/lessons.lesson11.SieveofEratosthenes.CountSemiprimes/eyabc.js

```diff
@@ -0,0 +1,38 @@
+function solution (N, P, Q) {
+  const semiCnt = new Array(N + 1).fill(0),
+    maxNum = N + 1;
+
+  const isPrime = (i) => {
+    for (let j = 2 ; j * j <= i ; j++)
+      if (i % j === 0) return false;
+    return true;
+  };
+
+  // 참고함
+  const isSemi = (num) => {
+    let cnt = 0;
+    for (let i = 2 ; cnt < 2 && i * i <= num ; ++i)
+      while (num % i === 0)
+        num /= i, ++cnt; // Increment count of prime numbers
+
+    // If number is greater than 1, add it to
+    // the count variable as it indicates the
+    // number remain is prime number
+    if (num > 1)
+      ++cnt;
+
+    // 나누어 지는 수가 2개 초과면 semi 가 아니다.
+    return cnt === 2;
```

Comment:
```suggestion
    return cnt;
```
이렇게 cnt만 반환해도 해결할 수 있을 것 같음!